Tests/SentryTests/SentryUIApplicationTests.swift

```diff
@@ -124,10 +121,16 @@ class SentryUIApplicationTests: XCTestCase {
     }
 
     private class MockSentryUIApplicationTests: SentryUIApplication {
-        weak var appDelegate: TestApplicationDelegate?
 
+        var notificationCenterWrapper = TestNSNotificationCenterWrapper()
```

Comment:
`m`: It's easy to mess if we have two initializers.

```suggestion
        let notificationCenterWrapper: TestNSNotificationCenterWrapper
```